app/components/publish-actions.tsx

```diff
@@ -38,7 +37,7 @@ export const PublishActions = (props: PublishActionProps) => {
 
 type TriggeredPopoverProps = {
   children: ReactNode;
-  renderTrigger: (
+  renderTrigger?: (
```

Comment:
@noahonyejese is there a case when this is actually undefined? By looking at the component name (`TriggeredPopover`), I'd assume that it should always be defined, otherwise the `Popover` can't be opened?

Reply:
Yes sorry, I'm still working on one thing that potentially will be undefined, and use a trigger prop without rendering anything for, issue #1758

Reply:
Right, but then maybe it could be solved with `{shouldRenderTriggeredPopover && <TriggeredPopover {...} />}` and not with making the thing that controls the popover state undefined, as then it would never be shown if I am not mistaken? 🤔 Maybe I still miss something, sorry 😅